xpdacq/md_schema_v1.json

```diff
@@ -0,0 +1,153 @@
+{
+  "type": "object",
+  "properties":{
+    "bt_piLast":{
+      "type": "string",
+      "description": "Last name of PI"
+    },
+    "bt_wavelength":{
```

Comment:
all the ``bt_sth`` are the metadata fields stored under ``Beamtime`` object and all of the fields will be dumped into ``mds`` per run.

wavelength might not be unique per beamtime and we actually allow it to be updated dynamically if needed